<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph2dVerificationSettingDialog</class>
 <widget class="QDialog" name="Graph2dVerificationSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Axis Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="xAxisGroupBox">
         <property name="title">
          <string>X Axis</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="xAxisTimeGroupBox">
            <property name="title">
             <string>Unit</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="xAxisTimeRadioButton">
               <property name="text">
                <string>Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="xAxisTimestepRadioButton">
               <property name="text">
                <string>Count</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="xAxisDistanceGroupBox">
            <property name="title">
             <string>Unit</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="xAxisDistanceRadioButton">
               <property name="text">
                <string>Distance</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="xAxisGridindexRadioButton">
               <property name="text">
                <string>Grid index</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="xAxisValueRangeGroupBox">
            <property name="title">
             <string>Value Range</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QCheckBox" name="xAxisValueRangeAutoCheckBox">
               <property name="text">
                <string>Auto</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="xAxisMinLabel">
                 <property name="text">
                  <string>Min:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="RealNumberEditWidget" name="xAxisMinEdit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="xAxisMaxLabel">
                 <property name="text">
                  <string>Max:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="RealNumberEditWidget" name="xAxisMaxEdit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xAxisLogCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Logarithmic axis</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="xAxisLabelLabel">
              <property name="text">
               <string>Label: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xAxisLabelEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="xAxisReverseCheckBox">
            <property name="text">
             <string>Reverse Direction</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="generalGroupBox">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="titleLabel">
              <property name="text">
               <string>Title: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="titleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="showTimeCheckBox">
            <property name="text">
             <string>Add Grid Indices etc. to Title</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="yAxisGroupBox">
         <property name="title">
          <string>Y Axis</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="yAxisSideComboBox">
              <item>
               <property name="text">
                <string>Left</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Right</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="yAxisValueRangeGroupBox">
            <property name="title">
             <string>Value Range</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="yAxisValueRangeAutoCheckBox">
               <property name="text">
                <string>Auto</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="yAxisMinLabel">
                 <property name="text">
                  <string>Min:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="RealNumberEditWidget" name="yAxisMinEdit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="yAxisMaxLabel">
                 <property name="text">
                  <string>Max:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="RealNumberEditWidget" name="yAxisMaxEdit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="yAxisLogCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Logarithmic axis</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="yAxisLabelLabel">
              <property name="text">
               <string>Label: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yAxisLabelEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="yAxisReverseCheckBox">
            <property name="text">
             <string>Reverse Direction</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/realnumbereditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Graph2dVerificationSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Graph2dVerificationSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yAxisValueRangeAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>yAxisMaxEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yAxisValueRangeAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>yAxisMinEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xAxisValueRangeAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>xAxisMinEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xAxisValueRangeAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>xAxisMaxEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xAxisValueRangeAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>xAxisLogCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yAxisValueRangeAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>yAxisLogCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
